docs/source/references/network_events.rst

```diff
@@ -0,0 +1,200 @@
+Network Events
+==============
+
+When there is an interaction with NuCypher smart contracts, various on-chain events are emitted. These events are
+defined in the :doc:`Contracts API </contracts_api/index>`, and they are queryable via the ``nucypher status events``
+CLI command, and allows for any NuCypher Network event to be queried.
+
+
+Querying Events
+---------------
+
+Since the number of events returned may be large, by default the query is limited to events within block numbers for the
+current period. However, this is configurable using the ``--from-block`` option.
+
+.. important::
+
+    Depending on the Ethereum provider being used, the number of results a query is allowed to return may be limited.
+    For example, on Infura this limit is currently 10,000.
+
+
+For a full list of CLI options, run:
+
+.. code::
+
+    $ nucypher status events --help
+
+
+For example, to view the staking rewards received by all Stakers in the current period, run:
```

Comment:
Turning these sections into linkable subheaders might make information easier to find for specific use cases.